SwiftMessages/SwiftMessagesHideKey.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  SwiftMessagesHideKey.swift
+//  SwiftMessages
+//
+//  Created by Mofe Ejegi on 11/05/2025.
+//  Copyright © 2025 SwiftKick Mobile. All rights reserved.
+//
+
+import SwiftUI
+
+private struct SwiftMessagesHideKey: EnvironmentKey {
+    /// By default, hide the current message.
+    static let defaultValue: @MainActor (Bool) -> Void = { animated in
```

Comment:
This isn't quite the same thing as the built-in `dismiss` environment variable, which is an instance of `DismissAction`. `DismissAction`'s is a struct, which prevents the behavior from being modified or overwritten (not to mention, the initializer isn't public). `DismissAction` that implements `callAsFunction()`, allowing the simple invocation `dismiss()`.

How about we make a struct `SwiftMessagesHideAction`:

1. `SwiftMessagesHideAction/callAsFunction()` calls `SwiftMessages.hide(true)`
2. An additional function `SwiftMessagesHideAction/hide(_ animated: Bool`) can be invoked if animation needs to be specified.